recipes/brian2/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "Brian2" %}
+{% set version = "2.1.3" %}
+{% set sha256 = "63bafb7198a252f7f74e80d74f6722e2a79be5643bdcd3e29145a7625b30e8ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython >=0.18
+    - setuptools >=6.0
+    # we build against the oldest supported numpy version
+    - numpy 1.10  # [not py36]
+    - numpy 1.11  # [py36]
+  run:
+    - python
+    # we don't need the same version as during the build, newer versions are
+    # still ABI-compatible
+    - numpy >=1.10  # [not py36]
+    - numpy >=1.11  # [py36]
+    - sympy >=0.7.6,!=1.1.0
+    - pyparsing
+    - scipy >=0.13.3
+    - weave  # [py2k]
+    - bsddb  # [py2k and not win]
+    - gsl >1.15
+    - cython >=0.18
```

Comment:
given that we need cython at runtime, do we also need compilers at runtime?

Reply:
> given that we need cython at runtime, do we also need compilers at runtime?

good point! It always worked fine on the test servers because they have system-wide installs of the necessary compilers, but it definitely makes sense to use the conda framework here as well.

I also noticed that my test would have actually passed even if runtime compilation with Cython failed, because we have a fallback solution with numpy in place (it would raise a warning but would not fail). I now explicitly ask for Cython which will raise an error if the compilation is not successful. Because things became too long for a single `python -c` line, the test is now in `run_test.py`.